### DISCUSSION TITLE
How to use katana crawler form post data

### DISCUSSION THREAD

**Discussion by j4vaovo** [projectdiscovery/katana#421] [Q&A] [answered]
<img width="627" alt="111" src="https://user-images.githubusercontent.com/128683738/229266468-7ec2f6b8-0224-40b3-898f-c02d8d5695e3.png">

**Comment by niudaii** [comment, marked as answer]
> <img alt="111" width="627" src="https://user-images.githubusercontent.com/128683738/229266468-7ec2f6b8-0224-40b3-898f-c02d8d5695e3.png">

![image](https://user-images.githubusercontent.com/41844046/234494476-181af35e-0b75-438b-abec-7dd9faa29c49.png)
<img width="852" alt="image" src="https://user-images.githubusercontent.com/41844046/234494782-7af15231-a3e3-4dcf-bfad-f875a05ee6c1.png">
